packages/manager/modules/telecom-dashboard/src/telecom-dashboard.scss

```diff
@@ -1,6 +1,6 @@
 .telecom-dashboard {
```

Comment:
This file should be renamed as `index.scss` in order to keep consistency across modules. Isn't it?